x/fundraising/keeper/auction.go

```diff
@@ -172,6 +185,22 @@ func (k Keeper) CreateBatchAuction(ctx sdk.Context, msg *types.MsgCreateBatchAuc
 
 	k.SetAuction(ctx, auction)
 
+	// Call hook after storing an auction
+	k.AfterBatchAuctionCreated(
+		ctx,
+		auction.Id,
+		auction.Auctioneer,
+		auction.StartPrice,
+		auction.MinBidPrice,
+		auction.SellingCoin,
+		auction.PayingCoinDenom,
+		auction.VestingSchedules,
+		auction.MaxExtendedRound,
+		auction.ExtendedRoundRate,
+		auction.StartTime,
+		auction.EndTimes[0],
+	)
```

Comment:
Isn't `MatchedPrice` needed? Even though it seems not necessary when created, `NewBatchAuction` has the following.
```
func NewBatchAuction(baseAuction *BaseAuction, minBidPrice sdk.Dec, matchedPrice sdk.Dec, maxExtendedRound uint32, extendedRoundRate sdk.Dec) *BatchAuction {
	return &BatchAuction{
		BaseAuction:       baseAuction,
		MinBidPrice:       minBidPrice,
		MatchedPrice:      matchedPrice,
		MaxExtendedRound:  maxExtendedRound,
		ExtendedRoundRate: extendedRoundRate,
	}
}
```

Reply:
Yes, it is used when a batch auction ends. `MatchedPrice` is not required in message level meaning that an auctioneer doesn't need to input the value to create a batch auction.